api/resolvers/item.js

```diff
@@ -568,6 +568,9 @@ export default {
         // extract id and create both links
         const matches = url.match(/(https?:\/\/)?((www\.)?(youtube(-nocookie)?|youtube.googleapis)\.com.*(v\/|v=|vi=|vi\/|e\/|embed\/|user\/.*\/u\/\d+\/)|youtu\.be\/)(?<id>[_0-9a-z-]+)/i)
         similar = `(http(s)?://)?((www.|m.)?youtube.com/(watch\\?v=|v/|live/)${matches?.groups?.id}|youtu.be/${matches?.groups?.id})((\\?|&|#)%)?`
+      } else if (urlObj.hostname === 'yewtu.be') {
+        const matches = url.match(/(https?:\/\/)?yewtu\.be.*(v=|embed\/)(?<id>[_0-9a-z-]+)/i)
+        similar = `(http(s)?://)?yewtu.be/(watch\\?v=|embed/)${matches?.groups?.id}((\\?|&|#)%)?`
```

Comment:
Also welcome to the team from me!

probably nitpick: I noticed that [yewtu.be](https://yewtu.be/) has a CNAME entry for www.yewtu.be:

```
$ dig +short www.yewtu.be
yewtu.be.
104.244.72.25
```

So this is missing a `(www\.)?` optional group like for YouTube links above.